sympy/assumptions/handlers/sets.py

```diff
@@ -388,6 +389,18 @@ def sin(expr, assumptions):
 
     cos, exp = [sin]*2
 
+    @staticmethod
+    def DenseMatrix(mat, assumptions):
```

Comment:
maybe you should put the superclass of matrices with entries?

Reply:
This will solve other reviews too. I will apply this suggestion.